### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ktiits**

### CODE REVIEW

**Review by samumantha — CHANGES_REQUESTED** [review]
Could we add the input/output and goal or at least just goal of each notebook to the contents?

File: README.md [hunk 1]
```diff
@@ -35,115 +35,75 @@ Terminology can be confusing , e.g. Cartographic label != ML label, please ask i
...
-    * counties dataset
-* Output:
-    * scaled and unscaled train, test validation datasets and labels
+This repository contains all Jupyter Notebooks and other code used in the course. Data is not inlcuded here, data download links are provided in data preparations Notebooks. Each exercise has its own folder:
 
-**03_raster_data_preparation**
+* [01_clustering](01_clustering)
+* [02_vector_data_preparation](02_vector_data_preparation)
+* [03_raster_data_preparation](03_raster_data_preparation)
+* [04_shallow_regression](04_shallow_regression)
```
Comment (samumantha) [inline README.md:95]:
```suggestion
* [04_shallow_regression](04_shallow_regression)
    * Runnning some shallow regression models on Paavo zip code dataset
    * Input:
        *  Scaled and unscaled train, test validation datasets and labels
    * Output:
        * Regression error metrics for validation dataset
```

File: README.md [hunk 2]
```diff
@@ -35,115 +35,75 @@ Terminology can be confusing , e.g. Cartographic label != ML label, please ask i
...
+* [05_shallow_classification](05_shallow_classification)
+* [06_deep_regression](06_deep_regression)
```
Comment (samumantha) [inline README.md:97]:
```suggestion
* [06_deep_regression](06_deep_regression)
    * Building and Runnning some deep regression models on Paavo zip code dataset
    * Input:
        *  Scaled train, test validation datasets and labels
```

File: README.md [hunk 3]
```diff
@@ -35,115 +35,75 @@ Terminology can be confusing , e.g. Cartographic label != ML label, please ask i
...
-* Input:
-    * Paavo zip code dataset
-    * counties dataset
-* Output:
-    * scaled and unscaled train, test validation datasets and labels
+This repository contains all Jupyter Notebooks and other code used in the course. Data is not inlcuded here, data download links are provided in data preparations Notebooks. Each exercise has its own folder:
 
-**03_raster_data_preparation**
+* [01_clustering](01_clustering)
+* [02_vector_data_preparation](02_vector_data_preparation)
```
Comment (samumantha) [inline README.md:93]:
```suggestion
* [02_vector_data_preparation](02_vector_data_preparation)
* Preparation of Paavo zip code dataset for machine learning
    * Input:
        * Paavo zip code dataset
        * Regions dataset
    * Output:
        * Scaled and unscaled train, test validation datasets and labels
```

Comment (ktiits) [reply README.md:93]: I would not add these here, these should be clearly in the beginning of each Notebook.
I do not like to have same things in many places -> sooner or later things will be not in sync any more..

**Review by samumantha — APPROVED** [review]

**Pull request merged**